sqlglot/dialects/duckdb.py

```diff
@@ -356,7 +356,6 @@ class Generator(generator.Generator):
                 if e.expressions and e.expressions[0].find(exp.Select)
                 else inline_array_sql(self, e)
             ),
-            exp.ArrayJoin: rename_func("ARRAY_TO_STRING"),
```

Comment:
DuckDB doesn't seem to have `array_join`. Is the translation to `array_to_string` handled elsewhere?

Reply:
Oh, you have an `ArrayToString` expression now, got it!

Reply:
Yep - refactored it since more dialects seem to have `ARRAY_TO_STRING` compared to `ARRAY_JOIN`.